app/jobs/appointment_notification/worker.rb

```diff
@@ -67,20 +67,6 @@ def send_message(notification, communication_type)
       raise UnknownCommunicationType, "#{self.class.name} is not configured to handle communication type #{communication_type}"
     end
 
-    log_info = {
```

Comment:
Instead of capturing the error and logging here, I'm just letting the twilio service fail, which will prompt a retry.

Reply:
I think the log is still useful. The Twilio error won't have context on what communication or notification ID, making it hard to diagnose when something does go wrong? I'd also be open to rescuing the exception, attaching this extra info to the exception, and re-raising the enriched exception. THis'll generate a rich error report that we can use to effectively diagnose. I'm not sure how easy it will be to attach context to the exception object itself though 😅